docs/source/architecture.md

```diff
@@ -10,11 +10,11 @@ Django is the heart of the application, which powers our backend. We use Django
 
 #### Django Rest Framework
 
-We use Django Rest Framework for writing and providing REST APIs. It's permission and serializers have helped write a maintainable codebase.
+We use Django Rest Framework for writing and providing REST APIs. Its permission and serializers have helped write a maintainable codebase.
 
 #### RabbitMQ
 
-We currently use RabbitMQ for queueing submission messages which are then later on processed by a Python worker.
+We currently use RabbitMQ to queue submissions for processing by a python worker.
```

Comment:
@mrMarce Please revert this change or change it such that it conveys the complete meaning.